MiniApp/Classes/Storage/MiniAppStatus.swift

```diff
@@ -1,10 +1,12 @@
 class MiniAppStatus {
     private let defaults: UserDefaults?
     private let miniAppInfoDefaults: UserDefaults?
+    private let miniAppCustomPermissionsDefaults: UserDefaults?
```

Comment:
should we use keychain for this? UserDefaults can easily be modified and allowing permissions is sensible. We can use a lib that makes the keychain behave like a UserDefault or just implement a helper class for it

Reply:
Permissions is something important/sensitive enough that it shouldn't be stored in userdefaults or anywhere in plaintext. Keychain sounds like a good option.

Reply:
Understood @Climbatize , @donnie-jp . Will make the changes 👍 . Thank you.